openapi.yml

```diff
@@ -4289,6 +4287,10 @@ paths:
                     in.c:
                       type: string
                       description: 'main: read'
+                componentAccess:
```

Comment:
Tohle nemá test, plus chybí test na to, že to spadne, když tam dám canManageTokens

Reply:
už jsou na oboje :-)